smallLibraries/Google.Cloud.Templates/content/gcpmvc/CSharp/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I don't seem to get this with "dotnet new mvc" - instead, libraries are preinstalled in wwwroot\lib.